src/main/webapp/js/transport.js

```diff
@@ -0,0 +1,153 @@
+
+function addFreightTransport(){
+
+    var transport = getFormData($("#add_transport_form"));
+    getAddress(transport.loadAddressId).done(function(result) {
+
+
+    	transport.loadAddress=result;
+        delete transport["loadAddressId"];
+        console.log(JSON.stringify(transport));
+
+    }).fail(function() {
+
+        console.log("Failed to get Address with id:"+transport.loadAddressId);
+        return -1;
+        });
+    getAddress(transport.unloadAddressId).done(function(result1) {
+
+
+    	transport.unloadAddress=result1;
+        delete transport["unloadAddressId"];
+        console.log(JSON.stringify(transport));
+
+
+    }).fail(function() {
+
+        console.log("Failed to get Address with id:"+transport.unloadAddressId);
+        return -1;
+        });
+    getClient(transport.clientId).done(function(result2) {
```

Comment:
Using more meaningful variable names would improve code readability ( e.g. `clientResult` instead of `result2`).